deploy.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash -eu
+
+# deploy_continuous_load - deploys the load injector and target application as well as
+# the ServiceMonitor and PodMonitor objects that are used to supply Prometheus with 
+# the scrape configuration.
+# Params:
+#  1) the namespace in which to deploy the monitors
+#  2) the target of the load generator. See https://k6.io/docs/javascript-api/k6-http/params/
+#     for into on accepted parameters
+#  3) the number of replicas to run
+#  4) the thresholds for pass/failure. See https://k6.io/docs/using-k6/thresholds/
+#  5) the number of requests per second to run
+# 
+# This implementation uses the podinfo application, which is able to generate 
+# metrics that are used in the dashboard.
+# 
+# This implementation uses k6 to generate the load.  The parameters are defined 
+# in continuous-load/charts/k6/load.js.
+function deploy_continuous_load() {
+  local namespace="$1"
+  local loadTargetService="$2"
+  local replicas="$3"
+  local thresholds="$4"
+  local reqPerSecond="$5"
+  echo "===> Deploying Continuous Load"
+  helm dependency build ./continuous-load
+  helm upgrade -install --wait continuous-load \
```

Comment:
can we make it do the helm chart just works without setting any parameters, so sensible default values, as a user of this isn't likely to take the shell script but integrate the helm chart into their existing path to production. So:

* A default namespace e.g. continuous-load
* A default for replicas, thresholds and rps